### ISSUE TITLE
Delete characters

### ISSUE THREAD

**Issue by PadPalon** [PadPalon/projectegon#18]
- [ ] Delete a existing character
- - [ ] Do not delete characters that are in active use

**Comment by Burki852** [comment]
Define in active use:
Thinking out loud: does that mean in a combat situation? Or could we build something like "Adventures" or "Campaigns" where characters could be added thus flagging them as active? Further discussions needed.

**Comment by PadPalon** [comment]
Yes, I've already created a Campaign entity, containing Users (as players or owners) and characters, since I had to find some kind of container to group users when logging in and checking rights. Thought this might be useful in the long run, or at the very least no bother us.

So with that I'd define "active" as any of these:
- Is in a campaign (these would be the player characters or other important characters)
- Is in an encounter (these would be the npcs)
